src/app/app.component.html

```diff
@@ -1,7 +1,25 @@
 <div class="content" role="main">
 
```

Comment:
We should not have empty rows in our HTML/template - Please add some code formatter (prettier, beautify etc.).